Sources/OpenAPIRuntime/EventStreams/ServerSentEventsDecoding.swift

```diff
@@ -28,9 +28,19 @@ where Upstream.Element == ArraySlice<UInt8> {
     /// The upstream sequence.
     private let upstream: Upstream
 
+    /// A closure that determines whether the given byte chunk should be forwarded to the consumer.
+    /// - Parameter: A byte chunk.
+    /// - Returns: `true` if the byte chunk should be forwarded, `false` if this byte chunk is the terminating sequence.
+    private let predicate: @Sendable (ArraySlice<UInt8>) -> Bool
+
     /// Creates a new sequence.
-    /// - Parameter upstream: The upstream sequence of arbitrary byte chunks.
-    public init(upstream: Upstream) { self.upstream = upstream }
+    /// - Parameters:
+    ///     - upstream: The upstream sequence of arbitrary byte chunks.
+    ///     - while: A closure that determines whether the given byte chunk should be forwarded to the consumer.
```

Comment:
```suggestion
    ///     - predicate: A closure that determines whether the given byte chunk should be forwarded to the consumer.
```

IIRC the parameter name (not the label) is what's used in docs - try to run the soundness script to verify.

Reply:
Yeah confirmed by CI:

```
Sources/OpenAPIRuntime/EventStreams/ServerSentEventsDecoding.swift:40:5: warning: [ValidateDocumentationComments] change the parameters of the documentation of 'init' to match its parameters
Sources/OpenAPIRuntime/EventStreams/ServerSentEventsDecoding.swift:65:9: warning: [ValidateDocumentationComments] change the parameters of the documentation of 'init' to match its parameters
Sources/OpenAPIRuntime/EventStreams/ServerSentEventsDecoding.swift:116:5: warning: [ValidateDocumentationComments] change the parameters of the documentation of 'asDecodedServerSentEventsWithJSONData' to match its parameters
```